pkg/import/dependency_descriptor.go

```diff
@@ -20,20 +20,25 @@ type DependencyDescriptor struct {
 	Kind                  string           `yaml:"kind"`
 	DefaultClusterStack   string           `yaml:"defaultClusterStack"`
 	DefaultClusterBuilder string           `yaml:"defaultClusterBuilder"`
+	Lifecycle             Lifecycle        `yaml:"lifecycle"`
 	ClusterStores         []ClusterStore   `yaml:"clusterStores"`
 	ClusterStacks         []ClusterStack   `yaml:"clusterStacks"`
 	ClusterBuilders       []ClusterBuilder `yaml:"clusterBuilders"`
 }
 
+type Source struct {
+	Image string `yaml:"image"`
+}
+
+type Lifecycle struct {
+	Source
+}
```

Comment:
```suggestion
type Lifecycle Source
```

Reply:
I think the suggested change may not allow the nesting we chose to use in the descriptor file (for consistency) - `lifecycle.image`
```
...
defaultStack: base
lifecycle:
  image: registry.pivotal.io/build-service/kpack-lifecycle@sha256:6ea2b62ccf3c3643417c78c7c3c4953f8291f54c0d0e725e2ecfc9ba23124a29
clusterStores:
...
```
Ref: [attached file](https://github.com/vmware-tanzu/kpack-cli/pull/149#issuecomment-769176860)

Reply:
😁  aye, works. making suggested change